test/addons-napi/test_array/test.js

```diff
@@ -19,19 +19,30 @@ const array = [
   ]
 ];
 
-assert.strictEqual(test_array.Test(array, array.length + 1),
-                   'Index out of bound!');
+assert.throws(
+  () => {
+    test_array.TestGetElement(array, array.length + 1);
+  },
+  /Index out of bounds!/
```

Comment:
Also fixed up the other instance which was there from the existing version of the test.